overlord/ifacestate/ifacestate_test.go

```diff
@@ -1865,7 +1865,7 @@ func (s *interfaceManagerSuite) TestStaleConnectionsIgnoredInReloadConnections(c
 }
 
 func (s *interfaceManagerSuite) testStaleAutoConnectionsNotRemovedIfSnapBroken(c *C, brokenSnapName string) {
-	s.mockIfaces(&ifacetest.TestInterface{InterfaceName: "test"})
+	s.mockIfaces(c, &ifacetest.TestInterface{InterfaceName: "test"})
```

Comment:
where does this change come from ?

Reply:
This was needed because `mockIfaces()` takes a *check.C as first argument in 2.52 but not in 2.53.

Reply:
And it's broken in our 2.52 tree :( one of the cherry picks broke it - oh well. so I fixed it here for expedience but maybe I should have done it in a separate PR

Reply:
Next time I think it would be useful to call it out in the PR description but thanks for explaining that